Gemfile

```diff
@@ -1,9 +1,11 @@
 source 'https://rubygems.org'
 
 gem 'syslogger', git: 'https://github.com/jbussdieker/syslogger.git', :branch => 'implement_log_formatter'
-gem 'rails', '4.2.5'
+gem 'rails', '4.2.8'
+gem 'rake', '< 11.0'
```

Comment:
Is there a reason this is `< 11.0` instead of `~> 11.0`?

Reply:
Yes, it cannot be 11.0 or later because of an issue with `Rake::Application` having an undefined method expected by another gem.

Reply:
Ah, okay, cool.